crates/cfsctl/src/main.rs

```diff
@@ -303,7 +303,7 @@ async fn main() -> Result<()> {
                     .as_ref()
                     .map(|p: &PathBuf| p.parent().unwrap())
                     .unwrap_or(Path::new("/sysroot"))
-                    .join("state")
+                    .join("state/deploy")
```

Comment:
Yup.  I agree: I noticed a while ago that I did the wrong thing here.  Thanks for fixing it.